test-integration/helpers/ui/user.coffee

```diff
@@ -94,7 +94,7 @@ class User extends TestHelper
     @el.loginSubmit().click()
 
   login: (username, password = 'password') =>
-    @el.loginLink().isDisplayed().then (isDisplayed) =>
```

Comment:
Is there a case for using `foo().isDisplayed()` instead of `foo().get().isDisplayed()`? If not, maybe it's worth removing?